### ISSUE TITLE
Reference missions

### ISSUE THREAD

**Issue by zorg2044** [CobaltWolf/Bluedog-Design-Bureau#820]
Issue to track and confirm if key missions can be performed by newly added LV's and spacecraft.

Testing in JNSQ (2.7x scale). Mandatory RCS, Engine ignitor preferred.

**Vanguard**
- [ ] Launch Vanguard 1,2,3 into orbit (spin decoupler not functioning as intended, delay between spin motors and decoupler staging not happening)

**Keyhole**
Launch into polar low orbit

- [ ] KH1 on Thor Agena A
- [ ] KH4 on Thor Agena B
- [ ] KH4B on Thrust Augmented Thor Agena B/D
- [ ] KH7 on Atlas SLV3 Agena D
- [ ] KH8 on Titan 23B
- [ ] KH9 on Titan III-D, 34D

**Redstone**

- [ ] Mercury redstone sub orbital hop (possible but trajectory quite steep, is this an issue?)
- [ ] Explorer I + Juno I into orbit
- [ ] WRESAT +Redstone sparta into orbit

**Juno Probes**
Juno II
- [ ] Explorer 7/8/11/S45/S46 into low orbit
- [ ] Pioneer 4 (reported by Friznit as possible but very challenging. Parking orbit not possible, direct injection from launch required.)

**Ranger**

Eve flyby
- [ ] Mariner 1& 2 - Atlas-LV3 Agena-B

Mun intercept, impact. With latest balance capturing into a highly elliptical orbit is barely possible though.
- [ ] Ranger Block 1 (Ranger 1/2) - Atlas-LV3 Agena-B
- [ ] Ranger Block 2 (Ranger 3/4/5) - Atlas-LV3 Agena-B
- [ ] Ranger Block 3 (Ranger 6-9) - Atlas-LV3 Agena-B

Nimbus

**Other Probes**
- [ ] Alouette
- [ ] Biosat
- [ ] Orbiting Frog Otolith - Scout B
- [ ] OGO
- [ ] OGO 1 - Atlas-LV3 Agena-B
- [ ] Luna Orbiter - Atlas-SLV3 Agena-D (Mun orbit)
- [ ] TIROS

**Pioneer Able (P3)**
Atlas LV3 Able. IRL profile unclear. In game the Atlas is able (heh) to put the Able upper stage into orbit on its own. The Able stage theoretically has enough Dv to make mun intercept. This makes the Star 20 altair extraneous technically. Furthermore the Star 20 does not have enough Dv to perform TLI on its own. However with engine ignitor the Able needs to be hotstaged calling for direct injection. 

Suggested profile is Direct injection using Atlas+Able into a highly elliptical orbit and complete TLI using the Star 20. P3 has enough Dv to make low munar orbit once injected.
- [ ] Pioneer P3 + Atlas Able

**Comment by friznit** [comment]
- Pioneer 3/4 - Juno II - Mun flyby into heliocentric orbit (requires a carefully timed launch for direct ascent and mun slingshot)
- Pioneer 6 - Delta E - Heliocentric orbit
- Explorer S-1, 7,8 (S30), 10 (S45), 11  - Juno II - these were launched into 50 deg inclined eccentric orbit
- Explorer 6 (S-2) - Thor Able - LKO
- IMP C - Delta C - also 50 deg incl eccentric orbit
- Nimbus were all polar LKO
- Surveyor - Atlas SLV-3 Agena-D - landed on Mun (can do Min as well no probs).
- OSO (Delta), OFO (Thor Delta), Tiros (Scout) - all LKO, though Scout can lob that thing into next week.

**Comment by CobaltWolf** [comment]
I think you flipped OFO and TIROS' launchers

**Comment by CobaltWolf** [comment]
I don't think there's an explorer 6 in the mod.

**Comment by CobaltWolf** [comment]
ack Surveyor is an Atlas Centaur not an Agena

**Comment by Morphisor244** [comment]
Vanguard is working fine, though needs a delicate ascent profile to make sure the final SRM stage starts at sufficient altitude to remain in orbit.

Juno II has sufficient delta-v and adequate staging to enter Mun orbit, didn't check for Kerbin escape yet.

Juno I is a rocket I feel is a little underpowered however. It manages orbit, but with not much margin. It was _just_ short of powerful enough to accomplish the Explorer-1 orbit parameters (according to History of spaceflight CP: Pe @ 100k, Ap @ 255k, inclination ~32,5). After at least 10 attempts with the best possible ascent, still came up short. It's probably less than 50 delta-v that's missing. I simply put in an extra solo-Sergeant on top to make it happen; those things have crazy TWR for the payload.

Of course, none of this is with Mandatory RCS & Engine ignitor, so may be of little value for your desired tests 🍭

**Comment by friznit** [comment]
Fixed my ramblings....

> I don't think there's an explorer 6 in the mod.
I mistook Pioneer 5 (P-2) for Explorer 6 (S-2) - they look remarkably similar.

**Comment by zorg2044** [comment]
> Juno I is a rocket I feel is a little underpowered however. It manages orbit, but with not much margin. It was _just_ short of powerful enough to accomplish the Explorer-1 orbit parameters (according to History of spaceflight CP: Pe @ 100k, Ap @ 255k, inclination ~32,5). After at least 10 attempts with the best possible ascent, still came up short. It's probably less than 50 delta-v that's missing. I simply put in an extra solo-Sergeant on top to make it happen; those things have crazy TWR for the payload.

@Morphisor244 Its worth noting that the actual historical reference mission for Explorer I should use the upgraded Juno I engine in the next tech node which uses hydyne fuel. With this config you can easily put the Explorer I into a 100x250km orbit just on the 11x+3x sergeant clusters. The best I could do with the Alcolox start config is something like 95x35km. Still it may be worth shaving 4-5kg from the Explorer probe just so that you can do it in the start node perhaps. @CobaltWolf thoughts?

**Comment by CobaltWolf** [comment]
U h yeah I never thought of that. Can you pay upgrade cost for something in the start node? Usually everything is unlocked by default

**Comment by zorg2044** [comment]
@CobaltWolf you can in fact have an upgrade with entry cost in the start node. Its very easy to miss though since the start node does not get that button to purchase all upgrades so unless you look at Start and the bottom of the list you wont see it. If we put it in start it should be without an upgrade perhaps as the second priority subtype.

**Comment by Morphisor244** [comment]
@zorg2044 You are right, I completely missed that. 2 reasons I did: it's really easy to miss in the build guide (updated one), since it's just the number 140/160 that's different. Also, like you mentioned, the upgrade is only unlocked at General rocketry. Should be at least Basic rocketry considering the timeframe and the fact it was just simultaneously with the less powerful version.
I would also suggest to add a little note accordingly to engine configs of the Sandstone engine.

Finally, the Sandstone 160 config results in 'only' 118 Delta-v added to the entire rocket compared to the 140 config, so it's still very marginal. Anything other than optimal ascent will still result in mission failure.

**Comment by zorg2044** [comment]
@Morphisor244 Have you tried that config? A big part of the problem with the alcolox engine is gravity losses due to low TWR. The higher thrust makes a bigger difference than the Isp bump. 

I got 95x35km after expending all stages with the alcolox engine but 105x1,000+ something with all 3 sergeants with the Hydyne engine. 105x250ish with only the first two sergeants. 

Its still a challenging mission in that you have to make sure you establish an Ap thats in space before MECO while trying to get as much horizontal velocity as possible but I dont think its super difficult like the Pioneer 1 profile.

**Comment by zorg2044** [comment]
Having discussed with Cobalt we're going to move the Hydyne config to the start node. Not much use for the engine outside of start anyway for most players. Hydyne will be default but people who want to be historically accurate can use the alcolox for Mercury and Jupiter C. Part description will have some pointers.

**Comment by Morphisor244** [comment]
Yeah I did try the hydyne config, it is indeed sufficient, but not as much room to spare; I couldn't get to 1000+ km like you, achieved 105x500km instead - mind the required inclination, correcting for that seems to sap some delta-v, could perhaps be more efficient there.

**Comment by zorg2044** [comment]
Yeah both my missions were at 35-45 degree inclination.

**Comment by jsolson** [comment]
In JNSQ the Mun and GEO are about 25% of Earth altitudes. We should adjust expectations accordingly. Discounting the first 95km since the atmospheres are about the same, the formula (OrbitAltKm - 95km) * 0.25 + 95km will give a close enough KSP adjusted target. So for Nimbus 1 it was expecting 937km at 98.5 degrees inclination. The adjusted altitude is 305km. I failed to get Numbus 1 to a 1000 x anything orbit. 305x305 at 98.5 degrees was a success.

FWIW the weight on Nimbus is good, maybe a touch light. Agena B and Thor DM19 balance also looks good. Thor is a little over fueled but it always under performed so I'm ok with that.

**Comment by zorg2044** [comment]
OGO eccentric profile verified with LV3 Agena B at 140x37,000km, 28 degrees (IRL was 300x150,000)

**Comment by Morphisor244** [comment]
> In JNSQ the Mun and GEO are about 25% of Earth altitudes. We should adjust expectations accordingly. Discounting the first 95km since the atmospheres are about the same, the formula (OrbitAltKm - 95km) * 0.25 + 95km will give a close enough KSP adjusted target. So for Nimbus 1 it was expecting 937km at 98.5 degrees inclination. The adjusted altitude is 305km. I failed to get Numbus 1 to a 1000 x anything orbit. 305x305 at 98.5 degrees was a success.
> 
> FWIW the weight on Nimbus is good, maybe a touch light. Agena B and Thor DM19 balance also looks good. Thor is a little over fueled but it always under performed so I'm ok with that.

While your logic seems sound, I find that correcting for the atmosphere in this way makes for very different orbital parameters than the original in any non-circular orbit.

Take the SCORE satellite for example. It had an orbit of 1484 x 185 km.
If we scale it down with your formula, it becomes (1484 - 95 ) * 0.25 + 95 = 442.25km Ap & (185 - 95 ) * 0.25 + 95 = 117,50 km Pe. The result being 442.25 x 117.50 km.
As you can see, ellipse of this rescaled orbit is vastly different and the required burn to get there is quite different as well.

This is why a straight 0.25 factor applied to the orbit distances seems to keep it better proportionate.
Of course, this still leaves us with the problem of dealing with the relatively high atmosphere, which becomes a problem.
For the same SCORE satellite, a straight 0.25 factor applied leads to an orbit of 371 x 46,25 km. Not sustainable in any scale of KSP.

I'm out of ideas here though 🤐

**Comment by zorg2044** [comment]
@Morphisor244 In my view JSO's suggested method of scaling the IRL orbits is Good Enough (tm). I think applying that method gives us reasonable goals for verifying the balance.

We're not doing RSS/RO and the aim is to get it into the right ballpark rather than recreating the mission that precisely.

**Comment by Morphisor244** [comment]
That is true.
I am solving this issue for historical contracts by simply not specifying a minimum periapsis when this would result in a value below ~140km after rescaling from the real world.
Instead, if the Ap is sufficiently high, I specify the rescaled Ap and add an approximation of the corresponding eccentricity it should have, as a minimum requirement. This way the shape of the orbit and the type of ascent needed for it are preserved regardless of scale.

**Comment by jsolson** [comment]
I tweaked the sergeant motors and the P3 probe so try Pioneer 4 and Pioneer Able P3 again. For launch timing a 120-130 degree phase angle to the Mun works out (MechJeb, target info).

It has a reaction wheel, but if P3 is supposed to be spin stabilized I've got no idea how it's supposed to orient for it's burns.

**Comment by CobaltWolf** [comment]
> It has a reaction wheel, but if P3 is supposed to be spin stabilized I've got no idea how it's supposed to orient for it's burns.

You're right. The orientation performed by the Able stage before spinning up the Altair was the last time the satellite is actively oriented. My understanding is... they just had a lot of women crunching numbers for days to find an exact ascent / injection profile that used the right amounts of delta V, that left the probe in an attitude where it could do its burns as needed.

**Comment by jsolson** [comment]
Well, I tried it a few times but I have no clue how to figure out the burn with the tools we have. We're going to have to call using the reaction wheel non-cheaty.

**Comment by dbandy13** [comment]
Regarding orbit scaling, what kind of balance do you get if you use JSOs formula to scale just the Pe and then apply the RL eccentricity to get an Ap?

**Comment by jsolson** [comment]
> Regarding orbit scaling, what kind of balance do you get if you use JSOs formula to scale just the Pe and then apply the RL eccentricity to get an Ap?

A 1000x400 km Earth orbit becomes a 308x171 km orbit scaling by Pe and applying the same eccentricity. 35786x250 km (GTO) becomes 9439x134 km, a bit higher than GTO, vs 9018x134 scaling both.

Earths radius is 6371 km, JNSQ is 1600 km. Orbital calcs don't use altitude so add that in.
`Eccentricity: e = (ap-pe)/(ap+pe)`
`Ratio: pe/ap = (1-e)/(1+e)`
`ap = pe / ratio`

The ratio of the Kerbin sma to the Earth sma doing it this way is always 26.19% It would be 25% if not for subtracting the 95 km.

:smile:

**Comment by dbandy13** [comment]
> 35786x250 km (GTO) becomes 9439x134 km, a bit higher than GTO, vs 9018x134 scaling both.

If you wanted to lock GTO to the correct alt and the ecc calc the Pe I get 8968x102 if my dV map is correct

**Comment by dbandy13** [comment]
> Well, I tried it a few times but I have no clue how to figure out the burn with the tools we have. We're going to have to call using the reaction wheel non-cheaty.

I've been thinking on this and I'm pretty sure if you pulled a 'perfect' hohmann to a retrograde lunar orbit it could be done. A full 180 degrees between departure/Mun periapsis should mean your initial prograde vector should be aligned with retrograde once you reach the mun. A little harder to achieve with direct injection though, I'm gonna try it soon

**Comment by zorg2044** [comment]
Actually tried out Pioneer 1 on Thor Able and it turned out to be easier than I thought. 
1) Thor Able is able to get it into low orbit
2) Altair has more than enough for a mun intercept
3) Killing it just short of Mun intercept and using the course correction motor allows for a very precise Mun intercept and ideal periapsis.
4) even being sloppy about phase angles able to complete the mission spin stabilized without reorienting. Insertion burn was a bit radial but burning a bit early, got it into an elliptical orbit.

**Comment by jsolson** [comment]
>Actually tried out Pioneer 1 on Thor Able and it turned out to be easier than I thought.

Even direct injection it's pretty easy for an experienced player. It's also very rewarding to make Mun orbit.

**Comment by zorg2044** [comment]
Pioneer 4 Juno II direct ascent to flyby also confirmed. Tried a 160 degree phase angle for launch.

**Comment by Morphisor244** [comment]
I know it's not on the list, but I launched Relay 1 today, using the Delta B build in accordance with the parts info and Friznit's wiki guide. Being the usual Thor lifter, Delta B 2nd stage with b9 switches set to Delta and the Altair set to X-248 config.
I flew using Mandatory RCS and playing by EngineIgnitor rules.

With this setup, I was unable to get the Relay probe into its historical orbit. Screenshot provided shows the intended orbit (see mission specs) and how much Dv I was still missing: https://i.imgur.com/inh2xV1.png
I'm sure the flight path could have been slightly better yet, but definitely not 250+ Dv to be gained in there.

I should note, I've had similar issues with all early Able/Delta probe launches - earlier I mostly put this down to building the probes too heavy. But this one is 100% historically correct, being an in-built replica. I'm not sure if it's down to the Able/Delta stage underperforming or the Altair. Thor seems fine, since Thor-Ablestar and Thor-Agena missions are perfectly doable.

**Comment by jsolson** [comment]
Scaled orbit for Relay 1 is 1935x401 km @ 45 deg. I'm getting well above that.

1st stage earliest RL-79. 2nd stage 118D. 3rd stage X-248.

![screenshot8](https://user-images.githubusercontent.com/16825376/86794861-f9372b80-c03a-11ea-8ce1-4eeef120e2dc.png)

![screenshot9](https://user-images.githubusercontent.com/16825376/86794935-0bb16500-c03b-11ea-964b-1deb8e5324e9.png)

![screenshot10](https://user-images.githubusercontent.com/16825376/86794981-1835bd80-c03b-11ea-8d7b-6dfb1458e5ee.png)

![screenshot11](https://user-images.githubusercontent.com/16825376/86794999-1e2b9e80-c03b-11ea-91ee-abad3ab9ce53.png)

**Comment by Morphisor244** [comment]
The BDB IMP parts are currently way heavier than their historical counterparts.
With both the probe core and the solar panels, antennas and magnetometer units added on, I got to 199 kg for IMP dry mass.
The real thing was about 60 kg only: https://space.skyrocket.de/doc_sdat/explorer_imp-a.htm
Or 100 kg for AIMP: https://space.skyrocket.de/doc_sdat/explorer_imp-d.htm

As a result, currently impossible to get them into correct orbit using the appropriate launcher (Delta C for IMP).

**Comment by zorg2044** [comment]
@Morphisor244 masses should be a little closer now. Havent done a flight test
